frontend/src/components/start/index.js

```diff
@@ -23,12 +26,16 @@ const Start = React.createClass({
     if (!params.token || !params.user_id) {
       throw 'Authentication failure - no token received'
     } else {
-      this.props.dispatch(signIn(params))
+      this.props.dispatch(signIn(params));
       this.props.router.push(`/showings`);
```

Comment:
We navigate before we've received login confirmation from backend. Trying to fetch data before we're logged in